<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>download_isc</class>
 <widget class="QWidget" name="download_isc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Download Catalogue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Output Dir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txt_dir"/>
             </item>
             <item>
              <widget class="QPushButton" name="btn_dir">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>output format</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txt_outfmt">
               <property name="text">
                <string>CATCSV</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>request</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="txt_request">
               <property name="text">
                <string>COMPREHENSIVE</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>search shape</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="txt_shape">
               <property name="text">
                <string>POLY</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>coord vals (deg)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="txt_coords"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>min dep (km)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="txt_min_dep">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>max dep (km)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="txt_max_dep">
               <property name="text">
                <string>1000</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>null dep</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="txt_null_dep"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>min mag</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="txt_min_mag">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>max mag</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLineEdit" name="txt_max_mag">
               <property name="text">
                <string>10</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>null mag</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="txt_null_mag"/>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>mag type(s)</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QLineEdit" name="txt_mag_type">
               <property name="text">
                <string>mb,M</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>mag agency(s)</string>
               </property>
              </widget>
             </item>
             <item row="11" column="1">
              <widget class="QLineEdit" name="txt_mag_ag">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>min year</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QLineEdit" name="txt_min_year">
               <property name="text">
                <string>1900</string>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>max year</string>
               </property>
              </widget>
             </item>
             <item row="13" column="1">
              <widget class="QLineEdit" name="txt_max_year">
               <property name="text">
                <string>2022</string>
               </property>
              </widget>
             </item>
             <item row="14" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>N. years (1 file)</string>
               </property>
              </widget>
             </item>
             <item row="14" column="1">
              <widget class="QLineEdit" name="txt_n_years">
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_apply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="MplCanvasMapWithToolbar" name="canvas_map" native="true"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Convert to OQ Catalogue</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_load_catalogues">
               <property name="text">
                <string>Load ISC Catalogues</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_convert">
               <property name="text">
                <string>Convert</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="list_catalogues"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_add_table_func">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_sub_table_func">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="table_func">
             <column>
              <property name="text">
               <string>Empirical Function</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Magnitude Type(s)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Min. Magnitude</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Max. Magnitude</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvasMapWithToolbar</class>
   <extends>QWidget</extends>
   <header>widgets.mplcanvas</header>
   <container>1</container>
   <slots>
    <slot>select_rectangle(bool)</slot>
    <slot>select_event(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_dir</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_dir_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_apply</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_apply_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_func</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>download_isc</receiver>
   <slot>_table_func_onCellChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_load_catalogues</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_load_catalogues_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_convert</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_convert_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_add_table_func</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_add_table_func_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sub_table_func</sender>
   <signal>clicked()</signal>
   <receiver>download_isc</receiver>
   <slot>_on_btn_sub_table_func_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_on_btn_dir_clicked()</slot>
  <slot>_on_btn_apply_clicked()</slot>
  <slot>_table_func_onCellChanged(int,int)</slot>
  <slot>_on_btn_load_catalogues_clicked()</slot>
  <slot>_on_btn_convert_clicked()</slot>
  <slot>_on_btn_add_table_func_clicked()</slot>
  <slot>_on_btn_sub_table_func_clicked()</slot>
 </slots>
</ui>
